docs/howtoguides/how_to_not_fix_related_usns.md

```diff
@@ -0,0 +1,65 @@
+# How to not fix related USNs
+
+When running the `pro fix` command for a USN, by default we also try to fix
+any related USNs as well (To better understand the concept of related USNs, please take 
+a look at this [document](.../references/what_are_related_usns.md). To make this clear,
+let's take a look into the following example:
```

Comment:
What was the command used to generate this output?

Reply:
That one: `sudo pro fix USN-5079-2` 

It is displayed near the beginning of the section `CVE/USN that require an Ubuntu Pro subscription`

Reply:
Hmm. I think that since we (elsewhere in the docs) always show any output with the command that produced it, we can justify repeating it here. It's good to be consistent, plus it reduces the potential for confusion.